### DISCUSSION TITLE
Spring Cloud AWS 3.0.0 general availability/release date

### DISCUSSION THREAD

**Discussion by Blockost** [awspring/spring-cloud-aws#399] [General]
The [maven repository](https://mvnrepository.com/artifact/io.awspring.cloud?sort=popular) does not contain the `v3.0.0` and the [docs](https://docs.awspring.io/spring-cloud-aws/docs/3.0.0-SNAPSHOT/reference/html/index.html#using-amazon-web-services) mention `3.0.0-SNAPSHOT` so I'm assuming  v3 has not been officially released yet ? 

Can you provide an expected release date ? I'd like to integrate spring-cloud-aws in an app that is using Spring Boot 2.7.0

**Comment by MatejNedic** [comment]
Hey @Blockost , 
we made docs available since 3.0.0 is under development and we wanted people to be able to see the progress.
Check [milestone](https://github.com/awspring/spring-cloud-aws/milestone/4) .

If you want to see [docs for 2.4.1 version](https://docs.awspring.io/spring-cloud-aws/docs/2.4.1/reference/html/index.html).

Link on 2.4.1 docs and 2.3.5 can also be found in readme.

**Reply by Blockost** [reply]
Hi @MatejNedic, thank you for your quick answer, it's a lot clearer now !
Keep up the good work 💪
